lib/shopping_cart.rb

```diff
@@ -0,0 +1,32 @@
+class ShoppingCart
+  attr_reader :name,
+              :capacity,
+              :products
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity.slice(0,2).to_i
+    @products = []
+    @shopping_cart_quantity = 0
+  end
+  def add_product(product)
+    @products << product
+    #@shopping_cart_quantity += product.quantity
```

Comment:
I think if you uncomment this line, and the `is_full?` method below, the method will work.